mapper/mapper.go

```diff
@@ -104,17 +104,15 @@ func CreateDataAggregationPage(cfg *config.Config, req *http.Request, basePage c
 // CreatePreviousReleasesPage maps type searchC.Response to model.Page
 func CreatePreviousReleasesPage(cfg *config.Config, req *http.Request, basePage coreModel.Page,
 	validatedQueryParams data.SearchURLParams, respC *searchModels.SearchResponse, lang string, homepageResponse zebedee.HomepageContent, errorMessage string,
-	navigationContent *topicModel.Navigation, template string, topic cache.Topic, validationErrs []coreModel.ErrorItem, zebedeeResp zebedee.PageData,
+	navigationContent *topicModel.Navigation, template string, topic cache.Topic, validationErrs []coreModel.ErrorItem, zebedeeResp zebedee.PageData, bc []zebedee.Breadcrumb,
 ) model.SearchPage {
 	page := model.SearchPage{
 		Page: basePage,
 	}
 
-	page.Metadata.Title = "Previous releases for " + zebedeeResp.Description.Title
+	page.Metadata.Title = zebedeeResp.Description.Title
```

Comment:
I'm not sure about this - I think the metatitle should still be 'Previous releases for .....' no?

Reply:
Ah but you're using it for the H1. Yes, so in which case we need to feed two different things I guess. Meta title <title> should be what you used to have here.

Reply:
Ugh the limitations of our page model. Our page model is a bit of a mess and should really get fixed

I've added a new `DisplayTitle` property to fix to the renderer to fix this. Waiting on the [PR](https://github.com/ONSdigital/dp-renderer/pull/167)

Reply:
👍